From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
Date: Fri, 4 Oct 2019 09:33:19 +0300
Subject: [PATCH] Disable server reloading


diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index 51a25d1ff6dbd1b630c7f1df0a5fc970098f8f0c..85a145f5e926206cddeda85d1c388c2efff7d1d3 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -757,6 +757,7 @@ public final class CraftServer implements Server {
 
     @Override
     public void reload() {
+        if (true) throw new UnsupportedOperationException(); // MikroFiber - disable server reloading
         org.spigotmc.WatchdogThread.hasStarted = false; // Paper - Disable watchdog early timeout on reload
         reloadCount++;
         configuration = YamlConfiguration.loadConfiguration(getConfigFile());
